index.html

```diff
@@ -124,82 +124,82 @@ <h2 class="text-center">My Journey</h2>
           <hr class="mg-btm" />
           <p>
             I started my life wanting to be a professional football player. When
-            i got scouted by VFL Wolfsburg i believed that my dream would come
+            I got scouted by VFL Wolfsburg I believed that my dream would come
             true. But after playing for VFL Wolfsburg for one year and for RB
-            Leipzig for 2 years, i had to accept the fact that i will not become
+            Leipzig for 2 years, I had to accept the fact that I will not become
             a major football player. Unsure on what is the right professional
-            fit for me i dived into several different paths of educations and
-            work. Until i finally came across programming and felt that this was
-            my passion. In the following months i consumed a lot of tutorials
+            fit for me I dived into several different paths of educations and
+            work. Until I finally came across programming and felt that this was
+            my passion. In the following months I consumed a lot of tutorials
             and courses online. Currently I am taking the bootcamp by neuefische
             and I am excited about what follows.
           </p>
         </div>
       </section>
 
-      <section class="bg-cl-yellow">
+      <section class="cv bg-cl-yellow">
         <div class="container">
           <h2 class="text-center">Experiences</h2>
           <hr class="mg-btm" />
           <h3>Educations</h3>
```

Comment:
If you only use an utility class once, like `bg-cl-yellow` I would encourage you to add it to the class `cv`. Also, the `div` with the class `container` is a bit unnecessary. You could either give the class `container` to the `section` and/or add it to the class `cv`.
```suggestion
      <section class="cv">
          <h2 class="cv__heading">Experiences</h2>
          <h3>Educations</h3>
```